### PR TITLE
Eng 8679

### PR DESCRIPTION
**Pull request by hhakimi53**

### CODE REVIEW

File: tests/frontend/org/voltdb/TestAdhocCreateDropIndex.java [hunk 1]
```diff
@@ -146,4 +134,127 @@ public void testBasicCreateIndex() throws Exception
...
+    public void testCreateDropIndexonView() throws Exception
+    {
+        VoltDB.Configuration config = new VoltDB.Configuration();
+        String ddl = "create table FOO (" +
+                     "ID integer not null," +
+                     "VAL bigint, " +
+                     "VAL1 float," +
+                     "constraint PK_TREE primary key (ID)" +
+                     ");\n" +
+                     "create table FOO_R (" +
```
Comment (wolffcm) [inline tests/frontend/org/voltdb/TestAdhocCreateDropIndex.java]: It's considered good form to keep the `partition table` statement close to the `create table` statement for the table that's being partitioned.

But it looks like here that the table `FOO_R` is not being used in your test?  In that case you can just remove that create table statement.

**Comment by wolffcm** [comment]
That's all my comments.

File: tests/frontend/org/voltdb/TestAdhocCreateDropIndex.java (hunk 1 above)
Comment (hhakimi53) [inline tests/frontend/org/voltdb/TestAdhocCreateDropIndex.java]: Done

**Comment by wolffcm** [comment]
Looks good to me.  Feel free to merge once you're sure that Jenkins is passing.

**Pull request merged**